src/pages/HomePage/index.vue

```diff
@@ -52,6 +54,10 @@ const tabs = computed<Tab[]>(() => [
     }[web3ProvidersStore.networkId],
   },
 ])
+
+const isTabsShown = computed(
+  () => route.name !== ROUTE_NAMES.appDashboardCapital,
+)
```

Comment:
I have only one special root that I check. It seems to me that it is not reasonable to write a meta for each root when only one of them will have a true flag

Reply:
You don't specify meta for route that doesn't require it. You'll have to do it for one route only in your case.

Also you never know what business need will be in future, and having only one special route now doesn't guarantee that it will stay like that forever.